### PR TITLE
TM-4134, hardcode values for Agenda Item create and update date

### PR DESCRIPTION
**Pull request by M00staff**
image

😺
Dual Merge: 
- [FE PR](TM-4030_create_update_date_ai)
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/771)

### CODE REVIEW

File: src/services/agendas.js [hunk 1]
```diff
@@ -249,7 +249,9 @@ const getAgendaItems = async (filsCols) => {
         aiperdetseqnum: ai.perdetseqnum,
         aipmiseqnum: pmi.pmiseqnum,
         aiitemcreatorid: ai.aiitemcreatorid,
+        aicreatedate: "2022-08-09T11:07:02",
         aiupdateid: ai.aiupdateid,
+        aiupdatedate: "2022-10-29T23:07:02",
```
Comment (SidelineCory24) [inline src/services/agendas.js:254]: double checking this was verified in dev1? if so disregard

Comment (M00staff) [reply src/services/agendas.js:254]: yes!
image

Comment (elizabeth-jimenez) [inline src/services/agendas.js:254]: don't have to hard code, there is already a value coming through in `ai` that you can pull from

Comment (elizabeth-jimenez) [reply src/services/agendas.js:254]: image

Comment (M00staff) [reply src/services/agendas.js]: added!

did someone add that recently - or did i completely miss it??

Comment (scott062) [reply src/services/agendas.js]: It was probably already there in the db columns but we weren't mapping it yet. I probably told you to hardcode it without first checking the AI object the ORM was returning before the mapping. You can console.log the `ai` in that mapping to see the object elizabeth printed

Comment (M00staff) [reply src/services/agendas.js]: prolly on me for not even checking lol

**Review by scott062 — APPROVED** [review]

**Pull request merged**